src/Actions/ManagesMessages.php

```diff
@@ -8,15 +8,16 @@ trait ManagesMessages
 {
 
     /**
-     * Get Messages [Come back here because of pagination]
+     * Get Messages
      *
-     * @return \Novu\SDK\Resources\Message
+     * @param array $queryParams
+     * @return mixed
      */
-    public function getMessages()
+    public function getMessages(array $queryParams = [])
     {
-        $response = $this->get("messages");
-
-        return new Message($response, $this);
+        $response = $this->get("messages", $queryParams);
+        $response['data'] = array_map(fn($value) => new Message($value, $this), $response['data']);
```

Comment:
This repo still allows PHP 7.2 and 7.3 to use it. So you can either use the normal function syntax instead of `fn` or bump the composer php version to start from `7.4`